test/fixtures/worldpay-3ds-flex-credentials.fixtures.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+const path = require('path')
+const _ = require('lodash')
+
+const pactBase = require(path.join(__dirname, '/pact-base'))
+const pactRegister = pactBase()
+
+const checkValidWorldpay3dsFlexCredentialsRequest = function checkValidWorldpay3dsFlexCredentialsRequest (opts = {}) {
+  const data = {
+    correlationId: opts.correlationId || 'a1',
+    gatewayAccountId: opts.gatewayAccountId || 333,
+    payload: {
+      organisationalUnitId: opts.organisationalUnitId || '5bd9b55e4444761ac0af1c80',
+      issuer: opts.issuer || '5bd9e0e4444dce153428c940',
+      jwtMacKey: opts.jwtMacKey || 'fa2daee2-1fbb-45ff-4444-52805d5cd9e0'
```

Comment:
This possibly needs to be `jwt_mac_key`

Reply:
Thanks. I have replaced `jwtMacKey` with `jwt_mac_key`.